cargo-shuttle/src/lib.rs

```diff
@@ -861,7 +859,7 @@ impl Shuttle {
     async fn local_run(&self, run_args: RunArgs) -> Result<()> {
         let services = self.pre_local_run(&run_args).await?;
         let (provisioner_server, provisioner_port) = Shuttle::setup_local_provisioner().await?;
-        let (logger_server, logger_port) = Shuttle::setup_local_logger().await?;
+        let (logger_server, _) = Shuttle::setup_local_logger().await?;
```

Comment:
I think we can complete remove the local logger now

Reply:
Should be covered here: [ed97c57](https://github.com/shuttle-hq/shuttle/pull/1204/commits/ed97c5706c664278afc8b3e06af5700c57a1e472).